<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-P8R83OF" failures="1" tests="2" name="Error Validation Test" time="51.65" errors="0" timestamp="2022-12-28T15:44:47 IST">
  <testcase classname="org.rahulshettyacademy.eCommerec_tc_2" name="FillForm_ErrorValidation" time="10.468">
    <failure type="java.lang.AssertionError" message="expected [Please enter your name:] but found [Please enter your name]">
      <![CDATA[java.lang.AssertionError: expected [Please enter your name:] but found [Please enter your name]
at org.rahulshettyacademy.eCommerec_tc_2.FillForm_ErrorValidation(eCommerec_tc_2.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- FillForm_ErrorValidation -->
  <testcase classname="org.rahulshettyacademy.eCommerec_tc_2" name="FillForm_PositiveFlow" time="24.88"/>
</testsuite> <!-- Error Validation Test -->
